docs/core/diagnostics/available-counters.md

```diff
@@ -16,27 +16,27 @@ The following counters are published as part of .NET runtime (CoreCLR) and are m
 
 | Counter | Description | First available in |
 |--|--|--|
-| :::no-loc text="% Time in GC since last GC"::: (`time-in-gc`) | The percent of time in GC since the last GC | .NET Core 3.1 |
```

Comment:
This feels more about when were they introduced rather than supported. Is this needed?

Reply:
Should we remove the column then?  It doesn't feel useful to know when a particular counter was introduced if it predates all available versions of .NET.

Reply:
I think it still might help clarify that these counters are not available on .NET Framework. It also preserves the structure of the table because some counters were introduced recently enough (.NET 7 for example) that we have supported versions that both do and do not have the counter. It would feel a little odd if some tables have the extra column but others don't depending on whether a counter was added in the last 3 years.